content/peloton/drone-google-chat/index.md

```diff
@@ -0,0 +1,128 @@
+---
+date: 2016-01-13T00:00:00+00:00
+title: Google chat
+author: josmo
+tags: [  notifications, chat  ]
+repo: josmo/drone-google-chat
+logo: gchat.png
```

Comment:
This is actually different the gtalk. It's more in the vain of slack then google talk.